skymap_scanner/utils/prepare_frames.py

```diff
@@ -67,18 +70,13 @@ def Process(self):
 
         self.PushFrame(frame)
 
-def prepare_frames(frame_array, baseline_GCD: Union[None, str], reco_algo: str, pulsesName: str) -> List[icetray.I3Frame]:
-    # type hint using list available from python 3.11
-    from icecube import (
-        DomTools,
-        VHESelfVeto,
-        dataclasses,
-        gulliver,
-        millipede,
-        photonics_service,
-        recclasses,
-        simclasses,
-    )
+def prepare_frames(frame_array, baseline_GCD: Union[None, str], reco_algo: str, pulsesName: str) -> List[icetray.I3Frame]: # type hint using list available from python 3.11
+
+    # ACTIVATE FOR DEBUG
+    # icetray.logging.console()
+
+    # Reconstruction algorithm provider class
+    RecoAlgo = recos.get_reco_interface_object(reco_algo)
```

Comment:
Does this return an instance or a class? The case convention here implies a class, but the function implies an instance

Reply:
Since we may want to add arguments to the RecoAlgo constructor in the future, this is supposed to be a class. I will rename `get_reco_interface_object` (it was returning a class from the beginning, after all).